integration/nemo/nemo_nvflare/config_sharer_sft.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from omegaconf import OmegaConf
+
+from nvflare.apis.client import Client
+from nvflare.apis.dxo import DXO
+from nvflare.apis.fl_constant import FLContextKey, ReturnCode
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.abstract.response_processor import ResponseProcessor
+
+from .constants import NemoDataKind
+
+
+class ConfigSharerSFT(ResponseProcessor):
```

Comment:
Do we need a separate config sharer for SFT? Can we just use [this](https://github.com/NVIDIA/NVFlare/blob/0ccfc653851fdc185e36d1084a07a17191883595/integration/nemo/nemo_nvflare/config_sharer.py#L29) one?

Reply:
let's keep it for now and unify the two in next round